semgrep-core/tests/OTHER/rules/prop_into_class.java

```diff
@@ -0,0 +1,7 @@
+static String secret = "secret"
+
+public class App
+{
+    // ruleid: prop-into-class 
+    String foo = sink(secret) 
```

Comment:
Where did this test come from? What are `static` variables declared outside classes? 🤔